remix-solidity/src/compiler/compiler.js

```diff
@@ -240,6 +240,10 @@ function Compiler (handleImportCall) {
     console.log('Loading ' + url + ' ' + (usingWorker ? 'with worker' : 'without worker'))
     self.event.trigger('loadingCompiler', [url, usingWorker])
 
+    if (worker !== null) {
+      worker.terminate()
+      worker = null
```

Comment:
Do we need to set this statement explicitly `worker = null`?

Reply:
yes, it is not null after termination.

Reply:
okay